File: app/src/main/java/cf/movie/slmovie/utils/rnUtils/toastDialog/ToastDialogReactPackage.java
package cf.movie.slmovie.utils.rnUtils.toastDialog;

import android.app.Activity;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

/**
 * Created by 包俊 on 2017/8/7.
 */

public class ToastDialogReactPackage implements ReactPackage {

    private Activity activity;

    public ToastDialogReactPackage(Activity activity) {
        this.activity = activity;
    }

    @Override
    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
        List<NativeModule> modules = new ArrayList<>();
        modules.add(new ToastDialogModule(reactContext, activity));
        return modules;
    }

    @Override
    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
        return Collections.emptyList();
    }
}

File: app/src/main/java/cf/movie/slmovie/utils/rnUtils/toastDialog/ToastDialogReactPackage.kt
package cf.movie.slmovie.utils.rnUtils.toastDialog

import android.app.Activity

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.NativeModule
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.uimanager.ViewManager

import java.util.ArrayList
import java.util.Collections

/**
 * Created by 包俊 on 2017/8/7.
 */

class ToastDialogReactPackage(private val activity: Activity) : ReactPackage {

    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
        val modules = ArrayList<NativeModule>()
        modules.add(ToastDialogModule(reactContext, activity))
        return modules
    }

    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
        return emptyList()
    }
}
